roles/download/tasks/prep_download.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: prep_download | Set a few facts (2)
+  set_fact:
+    download_force_cache: "{{ true if download_run_once else download_force_cache }}"
+
+- name: prep_download | Get facts
+  setup:
+
+- name: prep_download | Create staging directory on remote node
+  file:
+    path: "{{ local_release_dir }}/images"
+    state: directory
+    recurse: yes
+    mode: 0755
+    owner: "{{ ansible_ssh_user | default(ansible_user_id) }}"
+  when:
+    - ansible_os_family not in ["CoreOS", "Container Linux by CoreOS"]
+
+- name: prep_download | Create local cache for files and images
+  delegate_to: localhost
+  delegate_facts: no
+  run_once: true
```

Comment:
nit: could you please move the above 3 parameters under the module after `become`, so we can stay consistent?

Reply:
I took-a-look at the code, but I don't understand what you mean. Could you rephrase the request?

Reply:
```
 - name: prep_download | Create local cache for files and images
    file:
      path: "{{ download_cache_dir }}/images"
      state: directory
      recurse: yes
      mode: 0755
   become: false
   delegate_to: localhost
   delegate_facts: no
   run_once: true
```
this is what i meant, sorry if i wasn't clear

Reply:
Done.